Userland/Libraries/LibGfx/Matrix.h

```diff
@@ -36,13 +36,23 @@ class Matrix {
     {
     }
 
-    Matrix(Matrix const& other)
+    constexpr Matrix(Matrix const& other)
     {
-        __builtin_memcpy(m_elements, other.elements(), sizeof(T) * N * N);
+        *this = other;
     }
 
-    Matrix& operator=(Matrix const& other)
+    constexpr Matrix& operator=(Matrix const& other)
     {
+#ifndef __clang__
+        if (is_constant_evaluated()) {
```

Comment:
Can we document the issue with clang's constexpr parsing here? 

Is it a known clang issue? what version did you see errors/warnings in?

Reply:
Oh, this was an issue with GCC not treating memcpy as constexpr. I found an unconfirmed bug for it while I was doing this: 

https://gcc.gnu.org/bugzilla/show_bug.cgi?id=94082